docs/types/value-types.rst

```diff
@@ -65,10 +65,10 @@ This means that, for example ``~int256(0) == int256(-1)``.
 Shifts
 ^^^^^^
 
-The result of a shift operation has the type of the left operand, truncating the result to match the type.
+This means that contrary to checked arithmetic, if the result of a bit shift operation will not fit its type, the transaction will *not* revert.
 Right operand must be unsigned type. Trying to shift by signed type will produce a compilation error.
 
-- For positive and negative ``x`` values, ``x << y`` is equivalent to ``x * 2**y``.
+- For positive and negative ``x`` values, ``x << y`` is equivalent to ``x * 2**y`` run in unchecked mode.
```

Comment:
Nitpick, but that's (for the better or worse) it is referred to in other parts of the documentation:
```suggestion
- For positive and negative ``x`` values, ``x << y`` is equivalent to ``x * 2**y`` run in "unchecked" mode.
```